### PR TITLE
feat: add some users to the readme

### PR DESCRIPTION
**Pull request by p-shahi**

### CODE REVIEW

**Comment by p-shahi** [comment]
I see quic-go also lists GitHub stars in its list. Is that the criteria you apply? If so, perhaps 200 star minimum threshold?

**Comment by marten-seemann** [comment]
> If so, perhaps 200 star minimum threshold?

I think the quic-go cutoff is 1000 stars, with an exception made for Ooni.

**Comment by p-shahi** [comment]
> > If so, perhaps 200 star minimum threshold?
> 
> I think the quic-go cutoff is 1000 stars, with an exception made for Ooni.

The majority of projects on this list are below 1000 stars, so that would be too high of a cutoff. If we do use stars, let's do 250 so we can include Wasp/IOTA

**Comment by mudler** [comment]
That looks nice!

Friendly poke from here :wave: @marten-seemann @vyzo 

- https://github.com/mudler/edgevpn 
- https://github.com/kairos-io/kairos

File: README.md [hunk 1]
```diff
@@ -82,3 +82,23 @@ There's a few things you can do right now to help out:
 
 We test against and support the two most recent major releases of Go. This is
 informed by Go's own [security policy](https://go.dev/security).
+
+# Notable Users
+Some notable users of go-libp2p are:
+- [Kubo](https://github.com/ipfs/kubo) - The Go implementation of IPFS
```
Comment (BigLep) [inline README.md:88]:
```suggestion
- [Kubo](https://github.com/ipfs/kubo) - The original Go implementation of IPFS
```

**Comment by p-shahi** [comment]
@marten-seemann @MarcoPolo may I get a 👍 on this one?

File: README.md [hunk 2]
```diff
@@ -82,3 +82,24 @@ There's a few things you can do right now to help out:
 
 We test against and support the two most recent major releases of Go. This is
 informed by Go's own [security policy](https://go.dev/security).
+
+# Notable Users
+Some notable users of go-libp2p are:
+- [Kubo](https://github.com/ipfs/kubo) - The original Go implementation of IPFS
+- [Lotus](https://github.com/filecoin-project/lotus) - An implementation of the Filecoin Network
```
Comment (marten-seemann) [inline README.md:89]:
```suggestion
- [Lotus](https://github.com/filecoin-project/lotus) - An implementation of the Filecoin protocol
```

**Review by marten-seemann — APPROVED** [review]
Should we add Berty as well? Other than that, lgtm.

**Pull request merged**